pkg/platform/platform.go

```diff
@@ -0,0 +1,23 @@
+package platform
+
+import (
+	"runtime"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var (
+	// Architecture holds the runtime architecture of the process.
+	Architecture string
+	// OSType holds the runtime operating system type (Linux, …) of the process.
+	OSType string
+)
+
+func init() {
+	var err error
+	Architecture, err = GetRuntimeArchitecture()
```

Comment:
oh gosh, last thing and then i promise i'll merge: we don't need to export this, and I don't find it idiomatic to prefix with `Get`, so i suggest renaming this to `runtimeArchitecture()` but feel free to protest.

Other than that, it's good to go